WordPress/src/main/res/layout/my_site_fragment.xml

```diff
@@ -126,6 +126,24 @@
 
             </RelativeLayout>
 
+            <!--Plans-->
+            <RelativeLayout
+                android:id="@+id/row_plans"
+                style="@style/MySiteListRowLayout">
+
+                <ImageView
+                    android:id="@+id/my_site_plans_icon"
+                    style="@style/MySiteListRowIcon"
+                    android:src="@drawable/gridicons_clipboard" />
+
+                <org.wordpress.android.widgets.WPTextView
+                    android:id="@+id/my_site_plans_text_view"
+                    style="@style/MySiteListRowTextView"
+                    android:layout_toRightOf="@id/my_site_plans_icon"
+                    android:text="@string/plans" />
```

Comment:
I also notice that the web interface shows the type of plan ("Free", etc.) here. Is that going to be added later?